src/components/player/cue-bar.tsx

```diff
@@ -120,7 +129,10 @@ const NoteCue: React.FC<any> = ({
     actions.seek(cue.startTime)
     actions.pause()
     track('opened cue', {cue: cue.text})
-    !muteNotes && setPlayerPrefs({activeSidebarTab: 1})
+    // !muteNotes && setPlayerPrefs({activeSidebarTab: 1})
```

Comment:
commented this line out because it conflicts with the check I do (`if (activeSidebarTab === 1)`)

Reply:
the idea here is that we are "very forward" with the notes and autodisplay in the sidebar if they aren't muted, but if that needs to be removed that is fine too